nss-cohort-36/c36-class-website
# Jg style 1

# Description
Please run the json server and then run npm start on your terminal. you should be able to hover over each image and see the alternative picture
Closes #[Enter issue #]
## Type of change
Please delete options that are not relevant.
# Testing Instructions
Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout Jg-style-1
4. please run your json server on 5002 and run npm start on your terimnal as well
# Checklist:
